flakes.yaml

```diff
@@ -6,3 +6,8 @@
 # "test/new-e2e/tests/agent-platform/install-script"
 #   - "TestInstallScript/test_install_script_on_centos-79_x86_64_datadog-agent_agent_7"
 
+# TODO: https://datadoghq.atlassian.net/browse/CONTINT-4143
+test/new-e2e/tests/containers:  
+  - TestEKSSuite/TestCPU/metric___container.cpu.usage{^kube_deployment:stress-ng$,^kube_namespace:workload-cpustress$}
+  - TestEKSSuite/TestCPU
+  - TestEKSSuite
```

Comment:
[`TestCPU` doesn’t fail only when run from the `TestEKSSuite`, but also when run from `TestKindSuite`](https://app.datadoghq.com/ci/test-runs?query=test_level%3Atest%20%40test.service%3Adatadog-agent%20%40git.branch%3Amain%20%40test.suite%3A%22github.com%2FDataDog%2Fdatadog-agent%2Ftest%2Fnew-e2e%2Ftests%2Fcontainers%22%20%28%40test.name%3A%22TestKindSuite%2FTestCPU%2Fmetric___container.cpu.usage%7B%5Ekube_deployment%3Astress-ng%24%2C%5Ekube_namespace%3Aworkload-cpustress%24%7D%22%20OR%20%40test.name%3A%22TestEKSSuite%2FTestCPU%2Fmetric___container.cpu.usage%7B%5Ekube_deployment%3Astress-ng%24%2C%5Ekube_namespace%3Aworkload-cpustress%24%7D%22%29%20%40test.status%3Afail&agg_m=count&agg_m_source=base&agg_t=count&citest_explorer_sort=time%2Cdesc&currentTab=overview&eventStack=AgAAAY9ZOqZi5yRezQAAAAAAAAAYAAAAAEFZOVpPcVppQUFBbERIYkVoU2pud0NHbAAAACQAAAAAMDE4ZjU5NDEtNDFiZi00NTRjLWFmMzktYWUzOTllMTA1ZDYx&fromUser=true&index=citest&mode=sliding&saved-view-id=2236559&testId=AgAAAY9ZOqZi5yRezQAAAAAAAAAYAAAAAEFZOVpPcVppQUFBbERIYkVoU2pud0NHbAAAACQAAAAAMDE4ZjU5NDEtNDFiZi00NTRjLWFmMzktYWUzOTllMTA1ZDYx&trace=AgAAAY9ZOqZi5yRezQAAAAAAAAAYAAAAAEFZOVpPcVppQUFBbERIYkVoU2pud0NHbAAAACQAAAAAMDE4ZjU5NDEtNDFiZi00NTRjLWFmMzktYWUzOTllMTA1ZDYx&start=1713013123066&end=1715605123066&paused=false).

```suggestion
test/new-e2e/tests/containers:  
  - TestEKSSuite/TestCPU/metric___container.cpu.usage{^kube_deployment:stress-ng$,^kube_namespace:workload-cpustress$}
  - TestEKSSuite/TestCPU
  - TestEKSSuite
  - TestKindSuite/TestCPU/metric___container.cpu.usage{^kube_deployment:stress-ng$,^kube_namespace:workload-cpustress$}
  - TestKindSuite/TestCPU
  - TestKindSuite

```